tools/expotools/src/commands/UpdateVendoredModule.ts

```diff
@@ -51,6 +56,73 @@ const ANDROID_DIR = Directories.getAndroidDir();
 const PACKAGES_DIR = Directories.getPackagesDir();
 const BUNDLED_NATIVE_MODULES_PATH = path.join(PACKAGES_DIR, 'expo', 'bundledNativeModules.json');
 
+const ReanimatedModifier: ModuleModifier = async function (moduleConfig: VendoredModuleConfig, clonedProjectPath: string): Promise<void> {
+  const firstStep = moduleConfig.steps[0];
+  const androidMainPathReanimated = path.join(clonedProjectPath, 'android', 'src', 'main');
+  const androidMainPathExpoview = path.join(ANDROID_DIR, 'expoview', 'src', 'main');
+  const JNIOldPackagePrefix = firstStep.sourceAndroidPackage!.split('.').join('/');
+  const JNINewPackagePrefix = firstStep.targetAndroidPackage!.split('.').join('/');
+
+  const replaceHermesByJSC = async () => {
+    const nativeProxyPath = path.join(clonedProjectPath, 'android', 'src',
+                                   'main', 'cpp', 'NativeProxy.cpp');
+    const runtimeCreatingLineJSC = 'jsc::makeJSCRuntime();'
+    const jscImportingLine = '#include <jsi/JSCRuntime.h>';
+    const runtimeCreatingLineHermes = 'facebook::hermes::makeHermesRuntime();'
+    const hermesImportingLine = '#include <hermes/hermes.h>';
+
+    const content = await fs.readFile(nativeProxyPath, 'utf8');
+    let transformedContent = content.replace(runtimeCreatingLineHermes, runtimeCreatingLineJSC);
+    transformedContent = transformedContent.replace(new RegExp(hermesImportingLine, 'g'), jscImportingLine);
+
+    await fs.writeFile(nativeProxyPath, transformedContent, 'utf8');
+  }
```

Comment:
💅 Could you please make sure this file is prettified? I think as a variable assignment `replaceHermesByJSC` should be proceeded by `;`